nodes/vector/interpolation_mk3.py

```diff
@@ -104,8 +104,8 @@ def sv_init(self, context):
         self.inputs.new('SvVerticesSocket', 'Vertices')
         self.inputs.new('SvStringsSocket', 'Interval').prop_name = 't_in'
         self.outputs.new('SvVerticesSocket', 'Vertices')
-        self.outputs.new('SvVerticesSocket', 'Tanget')
-        self.outputs.new('SvVerticesSocket', 'Unit Tanget')
+        self.outputs.new('SvVerticesSocket', 'Tangent')
```

Comment:
`socket.name` renames should be avoided, they will break existing layouts in a very unpleasant way.

Reply:
Is there a way to softly deprecate them?